cadence/contracts/bridge/FlowEVMBridgeUtils.cdc

```diff
@@ -0,0 +1,401 @@
+import "NonFungibleToken"
+import "FungibleToken"
+import "FlowToken"
+
+import "EVM"
+import "FlowEVMBridgeConfig"
+
+/// This contract serves as a source of utility methods leveraged by FlowEVMBridge contracts
+//
+access(all) contract FlowEVMBridgeUtils {
+
+    /// Address of the bridge factory Solidity contract
+    access(all) let bridgeFactoryEVMAddress: EVM.EVMAddress
+    /// Delimeter used to derive contract names
+    access(self) let contractNameDelimiter: String
+    /// Mapping containing contract name prefixes
+    access(self) let contractNamePrefixes: {Type: {String: String}}
+
+    /// Returns an EVMAddress as a hex string without a 0x prefix
+    ///
+    /// @param: address The EVMAddress to convert to a hex string
+    ///
+    /// @return The hex string representation of the EVMAddress without 0x prefix
+    ///
+    // TODO: Remove once EVMAddress.toString() is available
+    access(all) fun getEVMAddressAsHexString(address: EVM.EVMAddress): String {
+        let addressBytes: [UInt8] = []
+        for byte in address.bytes {
+            addressBytes.append(byte)
+        }
+        return String.encodeHex(addressBytes)
+    }
+
+    /// Returns an EVMAddress as a hex string without a 0x prefix
+    ///
+    /// @param: address The hex string to convert to an EVMAddress
+    ///
+    /// @return The EVMAddress representation of the hex string
+    ///
+    access(all) fun getEVMAddressFromHexString(address: String): EVM.EVMAddress? {
+        let addressBytes: [UInt8] = address.decodeHex()
+        return EVM.EVMAddress(bytes: [
+            addressBytes[0], addressBytes[1], addressBytes[2], addressBytes[3],
```

Comment:
Left a comment about these further down

Reply:
Agreed, ideally we'll have utility methods embedded into the EVM contract & EVMAddress struct itself to handle this - see https://github.com/onflow/flow-go/issues/5422